tensorboard/components/lib.html

```diff
@@ -0,0 +1,21 @@
+<!--
+@license
+Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<link rel="import" href="analytics.html" />
```

Comment:
Do we need to factor this out?  I'm not a big fan of having essentially a proxy import just to avoid repetition; it's better for each file to explicitly import what it uses.  This also makes it more obvious when we are loading central things like analytics or polymer itself rather than obscuring them behind a generic "lib.html".

In that vein, does the `ng_tensorboard.html` actually need all of these?  I assumed it didn't use `tf-tensorboard` at all because it uses `tf-ng-tensorboard` instead.

Reply:
> In that vein, does the ng_tensorboard.html actually need all of these?

Yes. tf_ng_tensorboard is just a shell and it will use all the Polymer components defined in TensorBoard as is. In order to do that, we need to load the polymer library globally, load the default plugins, use the analytics as is defined here. What did not work with the traditional index.html is the fact that it rendered `<tf-tensorboard>` onto the body which resulted in "double rendering". In theory, we would not need tf-tensorboard but I am using that as a catch all entry point that depends on many tf_backend (runsStore, envStore) it depends on.

Reply:
I guess I'd still kind of prefer then to just inline them in both places, even if it's repetitive, rather than using this as a proxy import.  That way it's more in the spirit of include-what-you-use and makes it easier later to identify and remove unnecessary imports, rather than assuming that they always must be mirrored even down the road after that's no longer true.

Reply:
I made the change according to the feedback. Though I do have your approval, I'd like to hear if my change reflects your expectation before submitting it. Would you take please a look at it